Tests/FluentAssertions.Specs/Primitives/DateTimeOffsetAssertionSpecs.cs

```diff
@@ -364,6 +364,189 @@ public void
 
         #endregion
 
+        #region (Not) BeExactly
+
+        [Fact]
+        public void Should_succeed_when_asserting_datetimeoffset_value_is_exactly_equal_to_the_same_value()
+        {
+            // Arrange
+            DateTimeOffset dateTime = new DateTime(2016, 06, 04);
+            DateTimeOffset sameDateTime = new DateTime(2016, 06, 04);
+
+            // Act
+            Action act = () => dateTime.Should().BeExactly(sameDateTime);
+
+            // Assert
+            act.Should().NotThrow();
```

Comment:
🔧 If the assertion is not expected to throw anything, you don't need to explicitly use `NotThrow` (anymore).